.npmrc

```diff
@@ -1,2 +1,5 @@
+shamefully-hoist=true
 strict-peer-dependencies=false
-auto-install-peers=false
+auto-install-peers=true
```

Comment:
We would likely want to enable `auto-install-peers=true` in the `konnect-ui-apps` repository as well.

@ValeryG I don't remember why it's currently disabled, do you?

Reply:
with early versions of pnpm and this setting set to true build wasn't finding some dependent modules during the build. We definitely should try it again as I am pretty sure ppm is more mature now that it was a year ago... :)